deploy/helm/logstash/values.yaml

```diff
@@ -0,0 +1,221 @@
+logstashConfig:
+  logstash.yml: |
+    http.host: 0.0.0.0
+    xpack.monitoring.enabled: true
+    xpack.monitoring.elasticsearch.username: '${ELASTICSEARCH_USERNAME}'
+    xpack.monitoring.elasticsearch.password: '${ELASTICSEARCH_PASSWORD}'
+    xpack.monitoring.elasticsearch.hosts: ["http://elk-prod-1125755348.us-east-1.elb.amazonaws.com:80"]
+
+  log4j2.properties: |
+    logger.elasticsearchoutput.name: 'logstash.outputs.elasticsearch'
+    logger.elasticsearchoutput.level: 'debug'
+
+  pipelines.yml: |
+    - pipeline.id: main
+      path.config: "/usr/share/logstash/pipeline/ebrelayer.conf"
+    - pipeline.id: sifnode
+      path.config: "/usr/share/logstash/pipeline/sifnode.conf"
+
+  elasticsearch-ca.pem: |
+
+logstashPipeline:
+  logstash.conf: |
+
+  ebrelayer.conf: |
+    input {
+      file {
+        path => "/container_log/containers/ebrelayer-**.log"
+        type => "relayer_log"
+        start_position => "beginning"
+        sincedb_path => "/container_log/ebrelayer.sincedbpath"
+      }
+     }
+    filter {
+      json {
+        source => "message"
+        remove_field => ["message"]
+        add_field => { "cluster" => '{{ .Values.logstash.args.cluster }}' }
+      }
+    }
+    output {
+      elasticsearch {
+        hosts => ["http://elk-prod-1125755348.us-east-1.elb.amazonaws.com:80"]
+        user => '${ELASTICSEARCH_USERNAME}'
+        password => '${ELASTICSEARCH_PASSWORD}'
+        index => "ebrelayer"
+      }
+    }
+  sifnode.conf: |
+    input {
+      file {
+        path => "/container_log/containers/sifnode-**.log"
+        exclude => "sifnode-cli-**.log"
+        type => "sifnode_log"
+        start_position => "beginning"
+        sincedb_path => "/container_log/sifnode.sincedbpath"
+      }
+     }
+      filter {
+        mutate {
+          add_field => { "cluster" => '{{ .Values.logstash.args.cluster }}' }
+        }
+    }
+    output {
+      elasticsearch {
+        hosts => ["http://elk-prod-1125755348.us-east-1.elb.amazonaws.com:80"]
```

Comment:
do we want to setup a dns for this?

Reply:
Yep, once we've figured out the SSL issues we're seeing with logstash and elasticsearch. The kibana endpoint is actually available via a nice DNS entry already.